extensions/keycloak/deployment/src/main/java/io/quarkus/keycloak/KeycloakConfig.java

```diff
@@ -11,53 +11,53 @@
 import io.quarkus.runtime.annotations.ConfigRoot;
 import io.quarkus.runtime.annotations.DefaultConverter;
 
-@ConfigRoot(name = "keycloak")
+@ConfigRoot
 public final class KeycloakConfig {
 
     /**
      * Name of the realm.
      */
-    @ConfigItem(name = "realm")
+    @ConfigItem
     String realm;
 
     /**
-     * Name of the realm.
+     * Name of the realm key.
      */
     @ConfigItem(name = "realm-public-key")
     Optional<String> realmKey;
 
     /**
      * The client-id of the application. Each application has a client-id that is used to identify the application
      */
```

Comment:
The docs seems a bit off here. Or the name of the property. Or maybe it's just me :). Could you check what this one is used for?

Reply:
Seems to be good. The doc is lifted from https://www.keycloak.org/docs/latest/securing_apps/index.html#_java_adapter_config

Reply:
OK. Not excited about it but I suppose it's better to be consistent with the JSON config file.